.changeset/forty-ligers-teach.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/catalog-model': patch
+---
+
+Add labels to one of the example entities.
```

Comment:
It does not look like the examples dir is published as part of the package when I look at package.json so we can skip this changeset completely :)